src/test/enhetstester/kotlin/no/nav/familie/ba/sak/task/BehandleFødselshendelseTaskTest.kt

```diff
@@ -151,6 +153,36 @@ internal class BehandleFødselshendelseTaskTest {
         }
     }
 
+    @Test
+    fun `skal fullføre task uten videre oppfølgning hvis det kastes MidlertidigEnhetIAutomatiskBehandlingFeil`() {
+        // Arrange
+        val autovedtakStegService =
+            mockk<AutovedtakStegService>().apply {
+                every {
+                    kjørBehandlingFødselshendelse(any(), any(), any())
+                }.throws(MidlertidigEnhetIAutomatiskBehandlingFeil("MidlertidigEnhetIAutomatiskBehandlingFeil"))
+            }
+
+        // Act
+        val fødselshendelseTask =
+            BehandleFødselshendelseTask.opprettTask(
+                BehandleFødselshendelseTaskDTO(
+                    nyBehandling =
+                        NyBehandlingHendelse(
+                            morsIdent = randomFnr(),
+                            barnasIdenter = listOf(randomFnr()),
+                        ),
+                ),
+            )
+
+        // Assert
```

Comment:
Dette blir vel kanskje mer "act og assert" i ett da man kjører `.doTask(...)` 😇 

```suggestion
        // Act & assert
```

Reply:
Godt poeng, gikk på auto-pilot